mos_tests/environment/os_actions.py

```diff
@@ -148,6 +148,9 @@ def create_server(self, name, image_id=None, flavor=1, scenario='',
             helpers.wait(
                 lambda: self.get_instance_detail(srv).status == "ACTIVE",
                 timeout=timeout)
+            logger.info('the server {0} is {1}'.
```

Comment:
Lets do it like "logger.info('the server {0} is {1}'.format(" not "logger.info('the server {0} is {1}'."